packages/dev/core/src/Rendering/GaussianSplatting/gaussianSplatting.ts

```diff
@@ -323,6 +360,9 @@ export class GaussianSplatting {
             )
         );
 
+        /// set positions only once, no need to update on view changed
+        this._worker?.postMessage({ positions: this._positions.slice(0, this._vertexCount * 3) });

```

Comment:
Do we still need `this._positions` in the main thread afterwards? If not, you can pass it as a transferable buffer (second parameter of `postMessage`) to avoid the copy.